minecraft_servers/tornadomc/manifest.json

```diff
@@ -9,10 +9,10 @@
     "de"
   ],
   "social": {
-    "web": "https://forum.tornadomc.net",
+    "web": "https://tornadomc.net",
     "web_shop": "https://store.tornadomc.net",
     "tiktok": "@tornadomc.net",
-    "discord": "https://discord.gg/55PZcudMHE",
+    "discord": "https://discord.gg/857mVgfKXZ",
```

Comment:
Doesn't seem like Discord for server?
https://i.imgur.com/JcHE68p.png

Reply:
Fixed